test/fit_tests/tests/fileserver/README.md

```diff
@@ -0,0 +1,43 @@
+# External File Server(On-static) Test:
+
+This test validate the function of RackHD On-static file server. You need put an config file in the /config directory containing the test file url. The test date should write in jon format and save as fileserver_config.jon. The following is an example.

```

Comment:
Good catch, Json, forgive my typo :)
Do you think should I upload the fileserver_config.json in to github? I worried there are some internal links/server in this file